settings/artifacts.js

```diff
@@ -17,11 +17,6 @@ module.exports = {
       WitPriceFeedsDataLib: "WitPriceFeedsDataLib",
     },
   },
-  "polygon:amoy": {
```

Comment:
What has changed in polygon amoy?

Reply:
nothing really, possibly just upgraded to latest compilation